contract/truffle-config.js

```diff
@@ -30,6 +30,56 @@ module.exports = {
       port: 7545,
       network_id: "*",
     },
+    bsc: {
+      provider: () => {
+        return new HDWalletProvider(
+          process.env.PRIVATE_KEY,
+          process.env.BSC_RPC_URL
+        );
+      },
+      network_id: 56,
+      skipDryRun: true,
+    },
+    tbsc: {
+      provider: () => {
+        return new HDWalletProvider(
+          process.env.PRIVATE_KEY,
+          process.env.TBSC_RPC_URL
+        );
+      },
+      network_id: 97,
+      skipDryRun: true,
+    },
+    matic: {
+      provider: () => {
+        return new HDWalletProvider(
+          process.env.PRIVATE_KEY,
+          process.env.MATIC_RPC_URL
+        );
+      },
+      network_id: 137,
+      skipDryRun: true,
+    },
+    mumbai: {
+      provider: () => {
+        return new HDWalletProvider(
+          process.env.PRIVATE_KEY,
+          process.env.MUMBAI_RPC_URL
+        );
+      },
+      network_id: 80001,
+      skipDryRun: true,
+    },
+    xdai: {
+      provider: () => {
+        return new HDWalletProvider(
+          process.env.PRIVATE_KEY,
+          "https://rpc.xdaichain.com/"
```

Comment:
I think it would be better to unify it with the other ones and list it in the env file.